webpack.config.client.js

```diff
@@ -46,7 +46,9 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       // need full URL for dev server & HMR: https://github.com/webpack/docs/wiki/webpack-dev-server#combining-with-an-existing-server
       publicPath: START_DEV_SERVER
         ? `http://localhost:${webpackDevServerPort}/`
-        : `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}/`,
+        : `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${
+            process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH
+          }`,
```

Comment:
I know this is ugly but this is how prettier wants it :-/

Reply:
You can use `// prettier-ignore` to ignore the next node, source: https://prettier.io/docs/en/ignore.html

Reply:
Thanks Phil - I've moved it into a temp variable to hopefully make it more readable without banishing prettier.